docs/pages/deploy-a-cluster/deployments/aws-active-passive-deployment.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: "AWS Multi-Region Active-Passive Architecture Deployment"
+description: "Deploying a high-availability Teleport cluster in two AWS regions."
+---
+
+For mission-critical Teleport use cases, you may use the architecture described in this guide to achieve 
+an automatic failover across multiple AWS regions. This architecture keeps Teleport accessible with minimal 
+disruption during the event of an entire cloud provider region outage. While this example is for AWS, the 
+general architecture can apply to various cloud providers and self-hosted examples as well.
+
+In this architecture, the Auth and Proxy components and their networking components run parallelly in two 
+different cloud regions. These components utilize a global DynamoDB storage backend for the cluster state and audit 
+logs, while a copied S3 bucket is used for session recordings. The Route53 DNS failover is the control point to switch 
+between the active and passive clusters during the event of a regional outage. This document mainly focuses on Kubernetes,
+but same concepts applies with Virtual Machines/EC2 instances.
+
+## Overview
+
+![Diagram showing this Teleport
+architecture](../../../img/deploy-a-cluster/aws-multi-region-active-passive-ha-deployment.png)
+
+### Key Components
+- Two sets of Teleport Auth and Proxy components are deployed in separate regions. In this example, the us-west-1 (Primary)
+  region Teleport cluster is the Active cluster, and the us-east-1 (Secondary) region Teleport cluster is the Passive cluster.
+- All teleport cluster components are deployed as autoscaling groups in both regions.
```

Comment:
This isn't really true if we're using Kubernetes, as ASGs are an EC2 concept. There is no link between a Kubernetes `ReplicaSet` and an AWS autoscaling group.

Reply:
Yes, I will update that as ReplicaSets